releasenotes/notes/add-support-non-hermitian-op-aerpauliexpectation-653d8e16de4eca07.yaml

```
@@ -0,0 +1,5 @@
+features:
+  - |
+    Added the support of non-hermitian operators for AerPauliExpectation.
+    This was used in :class:'.QEOM' which could previously only be used with
+    PauliExpectation for qasm simulators.
```

I reformulated this a bit to make it less QEOM-centric 🙂 
```suggestion
    Added the support of non-hermitian operators for :class:`~qiskit.opflow.AerPauliExpectation`.
    This for instance enables using Aer's fast snapshot expectation value in algorithms such as
    :class:`~qiskit.algorithms.QEOM`.
```